### ISSUE TITLE
open podbit/db: no such file or directory

### ISSUE THREAD

**Issue by lsshawn** [ethanv2/podbit#1]
How do I change the directory for newboat cache.db? 

Am on v1.2.2.

**Comment by ethanv2** [comment]
Podbit respects the newsboat "download-path" setting in your newsboat config file. This is usually located in ".config/newsboat/config". For example, my setting is set to:

```
download-path ~/Downloads/Podcasts/
```

Until recently, podbit expects this directory to exist on startup, so you may need to create it.

**Comment by ethanv2** [comment]
I've found the actual error. It relates to handling of XDG_DATA_HOME being unset. I'll work on a fix - thanks for the report.

**Comment by ethanv2** [comment]
Fixed as of v1.3.1. Please upgrade to get the fix. Alternatively, put the following in your ".profile" file and the current version will function just fine:

```shell
export XDG_DATA_HOME="$HOME/.local/share"
```

AUR update coming out shortly. Thanks for the report, closing now.
